config/asset-merger.php

```diff
@@ -7,6 +7,7 @@
 		Assets::JAVASCRIPT => DOCROOT.'js'.DIRECTORY_SEPARATOR,
 		Assets::STYLESHEET => DOCROOT.'css'.DIRECTORY_SEPARATOR,
 	),
+	'show_paths' => TRUE,
```

Comment:
Thanks for keeping the default to be the current one so we don't have _breaking_ changes.